platform/src/abstractions/EducationBenchmarking.Platform.Domain/DataObjects/FinancialPlanDataObject.cs

```diff
@@ -49,4 +42,32 @@ public record FinancialPlanDataObject
     [JsonProperty("pupilsYear4")] public int? PupilsYear4 { get; set; }
     [JsonProperty("pupilsYear5")] public int? PupilsYear5 { get; set; }
     [JsonProperty("pupilsYear6")] public int? PupilsYear6 { get; set; }
+    [JsonProperty("pupilsYear7")] public int? PupilsYear7 { get; set; }
+    [JsonProperty("pupilsYear8")] public int? PupilsYear8 { get; set; }
+    [JsonProperty("pupilsYear9")] public int? PupilsYear9 { get; set; }
+    [JsonProperty("pupilsYear10")] public int? PupilsYear10 { get; set; }
+    [JsonProperty("pupilsYear11")] public int? PupilsYear11 { get; set; }
+    [JsonProperty("pupilsYear12")] public decimal? PupilsYear12 { get; set; }
```

Comment:
is there a reason of having pupilsYear12 and pupilsYear13 being decimal when rest are type int?

Reply:
Great spot - typo